hackathon_contributions_new_data_format/Isensee_JCB2018/visualization/main.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import getDataToBePlotted
+
+DataFilePath = "https://raw.githubusercontent.com/LoosC/Benchmark-Models/" \
+               "hackathon/hackathon_contributions_new_data_format/" \
+               "Isensee_JCB2018/measurementData_Isensee_JCB2018.tsv"
+
+ConditionFilePath = "https://raw.githubusercontent.com/LoosC/" \
+                    "Benchmark-Models/hackathon/hackathon_contributions_" \
+                    "new_data_format/Isensee_JCB2018/" \
+                    "experimentalCondition_Isensee_JCB2018.tsv"
+
+VisualizationFilePath = "https://raw.githubusercontent.com/LoosC/"\
+                        "Benchmark-Models/visualization/hackathon_contributions"\
+                        "_new_data_format/Isensee_JCB2018/visualizationSpecific"\
+                        "ation_Isensee_JCB2018.tsv"
+
+# import measurement data
+measurement_data = pd.DataFrame.from_csv(
```

Comment:
> FutureWarning: from_csv is deprecated. Please use read_csv(...) instead.

Consider using `petab.get_measurement_df(filename)`
and `petab.get_condition_df(filename)` below